codespell_lib/tests/test_dictionary.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+import os.path as op
+import re
+
+
+def test_dictionary_formatting():
+    """Test that all dictionary entries are in lower case and non-empty."""
+    err_dict = dict()
+    ws = re.compile(r'.*\s.*')  # whitespace
```

Comment:
I was going to say this will match just a single space, but that's not a bad thing anyway I guess...